src/view/components/common/login-as/login-as.ts

```diff
@@ -39,7 +36,7 @@ export class LoginAsComponent extends BaseComponent {
     dolLoginAs(options: any): void {
         this.loginService.loginAs(options).subscribe(data => {
             if (data) {
-                this.router.go('/dotCMS');
+                this.router.navigate(['dotCMS']);
```

Comment:
maybe we should create a wrapper for this router and have method like:

goToLogin or GoToMain